src/Components/ADT3DSceneBuilder/Internal/Behaviors/VisualRules/VisualRuleList.tsx

```diff
@@ -1,9 +1,12 @@
-import { IContextualMenuItem } from '@fluentui/react';
+import { IContextualMenuItem, Theme, useTheme } from '@fluentui/react';
+import Svg from 'react-inlinesvg';
 import React, { useEffect, useState } from 'react';
 import { TFunction, useTranslation } from 'react-i18next';
```

Comment:
Not sure if you added this `TFunction` import, but it showed me an error when I pulled this branch and looked at the code. The correct import seems to be `import { TFunction } from 'i18next';` and it is not a generic so the <string> besides it is not required.